README.md

```diff
@@ -74,6 +74,108 @@ The following types are exported:
+            "Content-Type": "application/json",
+          },
+        };
+      }
+    
+      // If so, we can make calls to the GitHub API - REST or GraphQL!
+      const MyOctokit = Octokit.plugin(restEndpointMethods);
+      const octokit = new MyOctokit({ auth: secrets.gitHub.bearerToken });
+    
+      // We'll list all open issues on the netlify/functions repository
+      const result = await octokit.rest.issues.list({
+        owner: "netlify",
+        repo: "functions",
+        state: "open",
+      });
+    
+      return {
+        statusCode: 200,
+        body: JSON.stringify(result),
+        headers: {
+          "Content-Type": "application/json",
+        },
+      };
+    };
+  ```
+
+### Checking additional metadata about auths in your functions
+
+Netlify Integration Auth Management also tracks metadata for the installed auth tokens. If you would like to verify that an auth has been installed with the correct scopes before calling into an API (say, for example, to give a better error message in the developer logs):
+
+  ```js
+    import { getSecrets } from "@netlify/functions";
+    
+    export const handler = async (event) => {
+      // Handle all fetching, refreshing, rotating, etc. of tokens
+      const secrets = await getSecrets();
+    
+      // We know that we need either "public_repo" or "repo" scopes granted
+      // in order to run this function properly
+      const sufficientScopes = ["public_repo", "repo"];
+    
+      // Secrets have an optional `grantedScopes` field that has details
+      // on what the auth is allowed to do
+      const tokenHasScope = secrets.gitHub.grantedScopes?.some((grantedScope) =>
+        sufficientScopes.includes(grantedScope.scope)
+      );
+    
+      // Notice how we can leave a great error message that tells us exactly what
+      // we need to do to fix things.
+      if (!tokenHasScope) {
+        return {
+          statusCode: 412,
+          body: JSON.stringify({
+            error: `You have enabled GitHub auth in your Netlify Auth dashboard, but it's missing a required scope. The auth must have one (or both) of the scopes: ${sufficientScopes.join(", ")}`,
+          }),
+          headers: {
+            "Content-Type": "application/json",
+          },
+        };
+      }
+    
+     // ...
+    }
+  ```
+
+### Accessing integration auth during development
+
+When running your site under `netlify dev`, the environmental variables that power the auth management will be synced, and you can transparently develop against the third party APIs - no additional configuration necessary!
+
+
+### Updating or removing auth from your site
+
+At any time you can revisit the Netlify Integration Auth Management tab for your site to see the installed auth. From there, you can select new scopes for already-installed auth and then run through the browser-based auth flow again, and the new scopes will be available to all your existing, deployed functions instantly. 
```

Comment:
This would benefit from being more specific about where the user can update items in the UI. (The location may change, however.)

Reply:
Included a link and instructions for the current location